moveit_ros/planning/rdf_loader/src/rdf_loader.cpp

```diff
@@ -66,14 +66,10 @@ RDFLoader::RDFLoader(const std::shared_ptr<rclcpp::Node>& node, const std::strin
 
   auto start = node->now();
 
-  urdf_string_ =
-      urdf_ssp_.loadInitialValue(node, ros_name, std::bind(&RDFLoader::urdfUpdateCallback, this, std::placeholders::_1),
```

Comment:
I'm missing the context a little, but why did you remove the callbacks?

Reply:
It looked to me like the callback was responsible for:
1. Saving the `urdf_string_`
2. Calling `loadFromStrings`

but those things are already done in the constructor, so I thought the callbacks could be safely removed.  I'm noticing now that the callbacks also call `new_model_cb_()` -- should this be added to the constructor as well?

Also, calling `loadFromStrings` from `urdfUpdateCallback`, before `srdf_string_` was loaded, produced an error message https://github.com/ros-planning/moveit2/blob/2ee83c31e107ff71f4469fe0d048eb18606c8e93/moveit_ros/planning/rdf_loader/src/rdf_loader.cpp#L106 which was harmless but a little confusing, and removing the callbacks got rid of that error message as well.